.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ jobs:
 
   image-tests-mpl302:
     docker:
-      - image: astropy/image-tests-py37-mpl302:1.10
+      - image: astropy/image-tests-py38-mpl302:1.10
```

Comment:
maybe switch directly to py39 instead of 38?

Reply:
@astrofrog pointed to https://docs.astropy.org/en/latest/development/testguide.html#running-image-tests but someone with server access still needs to generate and upload the new image(s).